lldb/test/API/functionalities/type_types/TestFindTypes.py

```diff
@@ -0,0 +1,69 @@
+"""
```

Comment:
There's also a unit-test for `GetTypeLookupContext` in `lldb/unittests/SymbolFile/DWARF/DWARFDIETest.cpp`. Would be nice to add a test-case there too (if the yaml isn't too much of a hassle)

Reply:
+1

Reply:
I can do that. I have a DWARF generator that I wrote in python.